src/Sylius/Behat/Context/Setup/PaymentRequestContext.php

```diff
@@ -55,7 +55,7 @@ public function thePaymentRequestActionHasBeenExecutedForOrderWithThePaymentMeth
      * @Given there is (also) a :state :action payment request for order :order using the :paymentMethod payment method
      */
     public function thePaymentRequestActionHasBeenExecutedForOrderWithThePaymentMethodAndState(
-        string $state = null,
+        ?string $state,
         string $action,
         OrderInterface $order,
         PaymentMethodInterface $paymentMethod,
```

Comment:
Nullable property should be placed after required properties.

```suggestion
        string $action,
        OrderInterface $order,
        PaymentMethodInterface $paymentMethod,
        ?string $state = null,
```

Reply:
Though why `$state` is nullable? If it's null, then `$this->getTransitionForState($state)` will be called and it will crash since this function requires state to be string.

Reply:
Apparently there error I fixed was fixed by another PR, the argument I changed was like that :
```php
string $state = null,
```
And this argument beeing the first one...

I will remove my commit since my fix is no longer needed, thank you for pointing it out 😉